test/end-to-end/client.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const Bluebird = require('bluebird');
+
+const path = require('path');
+const { createManagementClient } = require('../../bin/lib/contentful-client');
+
+const config = {
+  accessToken: process.env.CONTENTFUL_INTEGRATION_MANAGEMENT_TOKEN,
+  application: 'contentful.migration-cli.e2e-test/0.0.0'
+};
+
+const client = createManagementClient(config);
+
+const makeRequest = function (spaceId, requestConfig) {
+  requestConfig.url = path.join(spaceId, requestConfig.url);
+  return client.rawRequest(requestConfig);
+};
+
+const waitForJobCompletion = Bluebird.coroutine(function * (makeRequest, spaceId, jobId) {
```

Comment:
Would be cool to also share this code with the integration tests since it's somewhat duplicated

Reply:
good idea, will update